src/plugins/kibana_react/public/validated_range/validated_dual_range.tsx

```diff
@@ -92,22 +115,13 @@ export class ValidatedDualRange extends Component {
           fullWidth={fullWidth}
           value={this.state.value}
           onChange={this._onChange}
-          focusable={false} // remove when #59039 is fixed
```

Comment:
Looks like @majagrubic  just added `focusable` 11 days ago in https://github.com/elastic/kibana/pull/58122.

@majagrubic can you explain why it was added and if its ok to remove?

Reply:
It was introduced alongside a11y tests. See https://github.com/elastic/kibana/issues/59039 for more context. I suspect if you remove it, a11y tests will start failing again. The alternative might be disabling the specific a11y rule for now. cc @myasonik

Reply:
I added it back in since tests did indeed fail without it